### PR TITLE
Sky 1209 e2e logging in

### PR DESCRIPTION
**Pull request by meeh0w**
## Overview
* Adds E2E tests for login page and login flow

### CODE REVIEW

**Review by peterjan — APPROVED** [review]
perfect 👍

File: src/components/forms/SignUpForm.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const SignUpForm = ({ onSuccess, onFailure }) => (
     }}
   >
     {({ errors, touched, isSubmitting }) => (
-      <Form className="flex flex-col gap-4">
+      <Form className="flex flex-col gap-4" data-testid="SignUpForm">
```
Comment (interstella-5555) [inline src/components/forms/SignUpForm.js:65]: nit: I think html props usually follow kebab-case so I would call it "sign-up-form" which would feel more natural for me but if you feel good about different casing convention then let me know

example for kebab-case in a different library https://testing-library.com/docs/queries/bytestid/

File: cypress/e2e/login-page.test.js [hunk 2]
```diff
@@ -1,27 +1,89 @@
...
+
+    it("renders login form", () => {
+      cy.getByTestId("LoginForm").should("be.visible");
+    });
+
+    it("allows switching to sign up form", () => {
+      cy.getByTestId("SignUpLink").should("be.visible").click();
+
+      cy.location("pathname").should("eq", "/auth/registration");
+    });
```
Comment (kwypchlo) [inline cypress/e2e/login-page.test.js:20]: this is common mistake when working with integration tests :) 

https://docs.cypress.io/guides/references/best-practices#Creating-tiny-tests-with-a-single-assertion

**Review by interstella-5555 — CHANGES_REQUESTED** [review]

**Review by interstella-5555 — APPROVED** [review]

File: src/components/forms/SignUpForm.js (hunk 1 above)
Comment (meeh0w) [reply src/components/forms/SignUpForm.js:65]: Prop names in general follow a naming convention, but I never heard of a convention about values of HTML attributes (unless we're talking `id` or `class`, which are restricted in terms of allowed characters).

I think it's easier to have a convention where the main renderable element of the component has the same `data-testid` as the component's name.
If we find ourselves adding it on nested elements (e.g. an `input` deeper in `SignUpForm`), I'd consider using a different casing (although I always used pascal case and it worked great for me) :)

**Pull request merged**